src/utils/sortTask.ts

```diff
@@ -1,7 +1,7 @@
 import { TaskResponse } from '@/types/dto/tasks.dto'
 
 export const sortTaskByDescendingOrder = (tasks: TaskResponse[]) => {
-  return tasks.sort((a, b) => {
+  return (tasks ?? []).sort((a, b) => {
```

Comment:
If `tasks` is typed as `TaskResponse[]`, why would we have to check for `tasks ?? []`? If this is meant for error handling, we should be better off throwing / catching errors here instead of rendering empty tasks imo.

Reply:
fixed in the commits below